<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppManageDialog</class>
 <widget class="QDialog" name="AppManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>651</width>
     <height>311</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>index</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>应用名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>启动方式</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>评分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>添加日期</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>           用户名称:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="userEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>           用户密码:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="passEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addButton_4">
      <property name="text">
       <string>更改</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>     应用名称:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>     启动方式:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="pathEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>添加应用</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addButton_2">
      <property name="text">
       <string>启动应用</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="delButton">
      <property name="text">
       <string>删除应用</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
